src/main/java/com/sparta/hirello/primary/card/entity/Card.java

```diff
@@ -50,4 +48,27 @@ public class Card {
 
     @OneToMany(mappedBy = "card")
```

Comment:
```suggestion
    @OneToMany(mappedBy = "card", cascade = CascadeType.ALL, orphanRemoval = true)
```
이렇게 부모 엔티티와 생명주기를 같게 만들어주세요.